docs/_static/tight_table.css

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
a new css file that takes a large table and forces it to fit on the page without the need for a horizontal scroll bar.